<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWidget</class>
 <widget class="QWidget" name="ChatWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
    border: 0px;
}
</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="chatWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>431</width>
       <height>501</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>431</width>
       <height>501</height>
      </size>
     </property>
     <widget class="QLabel" name="friendNameLabel">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>1</y>
        <width>121</width>
        <height>62</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>432</width>
        <height>65</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(190, 190, 190);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="transPushButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>373</y>
        <width>31</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>传</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>365</y>
        <width>432</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(202, 202, 202);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTextEdit" name="chatTextEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>403</y>
        <width>431</width>
        <height>97</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="sendPushButton">
      <property name="geometry">
       <rect>
        <x>356</x>
        <y>476</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>发  送(S)</string>
      </property>
     </widget>
     <widget class="ChatListView" name="chatListView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>65</y>
        <width>431</width>
        <height>300</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
     </widget>
     <zorder>label_4</zorder>
     <zorder>label_5</zorder>
     <zorder>chatTextEdit</zorder>
     <zorder>sendPushButton</zorder>
     <zorder>transPushButton</zorder>
     <zorder>chatListView</zorder>
     <zorder>friendNameLabel</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChatListView</class>
   <extends>QListView</extends>
   <header>chatlistview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
